package.json

```diff
@@ -111,5 +111,6 @@
       "packages/**"
     ]
   },
-  "dependencies": {}
+  "dependencies": {},
+  "packageManager": "yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e"
```

Comment:
I think this needs to be removed.

Reply:
I think that should be okay? my understanding is that the package manager is used if corepack is enabled so corepack will know which version of yarn to run, and now anyone who develops in PF-react should have corepack enabled. This patch release branch did not have the yarn upgrade cherry picked onto it, so i would expect to see this when someone builds pf-react locally on this branch.